<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>What is MetaKG - MetaKG</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "What is MetaKG";
        var mkdocs_page_input_path = "about.md";
        var mkdocs_page_url = null;
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> MetaKG
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Welcome to MkDocs</a>
                </li>
              </ul>
              <ul class="current">
                <li class="toctree-l1 current"><a class="reference internal current" href="./">What is MetaKG</a>
    <ul class="current">
    </ul>
                </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">MetaKG</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
      <li class="breadcrumb-item active">What is MetaKG</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <h1 id="what-is-metakg">What is MetaKG</h1>
<p>MetaKG is the abbreviation of Metabolite-centric Knowledge Graph. It is a knowledge graph that is centered around metabolites and their associations with other entities such as genes, proteins, diseases, and pathways. MetaKG is a multi-modal knowledge graph that integrates information from various sources including databases, literature, and ontologies. MetaKG is designed to be a comprehensive and up-to-date resource for metabolite-centric knowledge.</p>
<h1 id="what-is-the-source-of-metakg">What is the source of MetaKG</h1>
<p>MetaKG is constructed by integrating information from various sources including databases, literature, and ontologies. The sources of MetaKG include but are not limited to the following:
- <strong>HMDB</strong> (The Human Metabolome Database) - HMDB is a comprehensive, web-accessible metabolite database. It contains detailed information about small molecule metabolites found in the human body.
- <strong>KEGG</strong> (Kyoto Encyclopedia of Genes and Genomes) - KEGG is a collection of databases dealing with genomes, biological pathways, diseases, drugs, and chemical substances.
- <strong>SMPDB</strong> (Small Molecule Pathway Database) - SMPDB is an interactive, visual database containing more than 600 small molecule pathways found in humans.
- <strong>ChEBI</strong> (Chemical Entities of Biological Interest) - ChEBI is a database and ontology of chemical entities of biological interest.
- <strong>PubChem</strong> - PubChem is a database of chemical molecules and their activities against biological assays.
- <strong>FooDB</strong> - FooDB is the world's largest and most comprehensive resource on food constituents, chemistry, and biology.
- <strong>UniProt</strong> - UniProt is a freely accessible database of protein sequence and functional information.</p>
<p>We use the data from these sources to construct MetaKG. We also use the literature to extract some information about metabolite associations from scientific articles both manually and automatically. HMDB, KEGG and SMPDB are the primary sources of metabolite information in MetaKG. We use the data from these sources to construct the MetaKG.</p>
<h1 id="what-is-the-purpose-of-metakg">What is the purpose of MetaKG</h1>
<p>MetaKG stands as an indispensable bastion of metabolite-centric knowledge, meticulously curated and continuously updated to serve as the go-to wellspring for researchers, developers, and data scientists alike. Its robust architecture empowers users to delve deep into the intricate realm of metabolomics, facilitating a spectrum of endeavors, including but not limited to:</p>
<ol>
<li><strong>Metabolite-centric Knowledge Graph Construction</strong>: Seamlessly construct comprehensive knowledge graphs elucidating the intricate relationships between metabolites and their biological contexts.</li>
<li><strong>Metabolite-centric Statistical Analysis</strong>: Harness the power of statistical methods tailored specifically for metabolomic data, unveiling patterns, trends, and insights that drive groundbreaking discoveries.</li>
<li><strong>Metabolomic and Biological Search</strong>: Navigate vast repositories of metabolomic and biological data with precision and efficiency, uncovering relevant information crucial for advancing research and understanding complex biological systems.</li>
<li><strong>Metabolite Association Analysis</strong>: Unravel the intricate web of associations between metabolites, diseases, genetic factors, and environmental influences, shedding light on underlying mechanisms and potential therapeutic targets.</li>
<li><strong>Metabolite-centric Entity Representation and Visualization</strong>: Transform raw data into intuitive visualizations and representations, facilitating comprehension and communication of complex metabolite-centric concepts.</li>
<li><strong>Metabolite-centric Machine Learning and Deep Learning</strong>: Leverage cutting-edge machine learning and deep learning algorithms tailored for metabolomic data analysis, unlocking predictive insights and accelerating discovery.</li>
<li><strong>Knowledge Graph Completion and Prediction</strong>: Enhance existing knowledge graphs and predict novel associations and interactions, paving the way for hypothesis generation and experimental validation.</li>
<li><strong>High-Quality Hypothesis Generation and Validation</strong>: Empower hypothesis generation with a foundation built upon comprehensive metabolite-centric knowledge, facilitating rigorous validation and driving scientific progress.</li>
<li><strong>And Beyond</strong>: Constantly evolving to meet the dynamic needs of the scientific community, MetaKG stands ready to support a myriad of endeavors aimed at unraveling the mysteries of metabolomics and advancing our understanding of biological systems.</li>
</ol>
<p>With MetaKG as their compass, researchers, developers, and data scientists embark on a journey of discovery, armed with the insights and tools needed to navigate the intricate landscape of metabolite-centric knowledge.</p>
<h1 id="how-can-you-use-metakg">How can you use MetaKG</h1>
<p>MetaKG is available as a website and a python package. You can use the website to search for metabolites and their associations with other entities. You can also use the python package to access the data programmatically. The python package provides an easy-to-use API to access the data in MetaKG. You can use the python package to construct knowledge graphs, train machine learning models, and perform statistical analysis on metabolite-centric data.</p>
              
            </div>
          </div><footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="Footer Navigation">
        <a href=".." class="btn btn-neutral float-left" title="Welcome to MkDocs"><span class="icon icon-circle-arrow-left"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
      <span><a href=".." style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
